howto make installcheck

### What happened?

Hi,

After  successful ```  make HAVE_MSGPACK=1,  make install ```  based postgres18 stable(personal build), then `make installcheck`,
however i got some regression cases failed.

```

# +++ regress install-check in  +++
# using postmaster on Unix socket, default port
not ok 1     - compatibility/schema/jsonb/script/element/string/seqscan       43 ms
not ok 2     - compatibility/schema/jsonb/script/element/boolean/seqscan       55 ms
not ok 3     - compatibility/schema/jsonb/script/element/number/seqscan       62 ms
not ok 4     - compatibility/schema/jsonb/script/value/string/seqscan       53 ms
not ok 5     - compatibility/schema/jsonb/script/value/boolean/seqscan       60 ms
not ok 6     - compatibility/schema/jsonb/script/value/number/seqscan       71 ms
ok 7         - compatibility/schema/jsonb/script/object/seqscan       66 ms

ok 36        - compatibility/schema/jsonb/script-v2/object/seqscan       60 ms
ok 37        - compatibility/schema/jsonb/match-v2/element/seqscan       62 ms
ok 38        - compatibility/schema/jsonb/match-v2/value/seqscan       53 ms
ok 39        - compatibility/schema/jsonb/query-v2/element/seqscan       52 ms
ok 40        - compatibility/schema/jsonb/query-v2/value/seqscan       53 ms
not ok 1572  - vacuum/two-phase-commit                   173 ms
ok 1573      - index-scan/kill-prior-tuple               137 ms
ok 1574      - int4-array/equal/indexscan                173 ms
ok 1575      - int4-array/equal/bitmapscan               106 ms
1..1575
# 1029 of 1575 tests failed.
# The differences that caused some tests to fail can be viewed in the file "/home/highgo/works/repo/pgroonga/regression.diffs".
# A copy of the test summary that you see above is saved in the file "/home/highgo/works/repo/pgroonga/regression.out".
make[1]: *** [/home/highgo/works/repo/postgres/inst/lib/pgxs/src/makefiles/pgxs.mk:433: installcheck] Error 1
make[1]: Leaving directory '/home/highgo/works/repo/pgroonga'
make: *** [Makefile:29: installcheck] Error 2
highgo@qd:~/works/repo/pgroonga$
highgo@qd:~/works/repo/pgroonga$ psql --version
psql (PostgreSQL) 18.1
highgo@qd:~/works/repo/pgroonga$
highgo@qd:~/works/repo/pgroonga$ git branch
* (HEAD detached at 4.0.4)
  main
```


Did i miss something?
Thanks in advance.

### How to reproduce it

`make installcheck`


### Expected behavior

regression cases OK

### OS and PostgreSQL version

OS: 
```
 cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 13 (trixie)"
NAME="Debian GNU/Linux"
VERSION_ID="13"
VERSION="13 (trixie)"
VERSION_CODENAME=trixie
DEBIAN_VERSION_FULL=13.0
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

postgres 18 :
```
 ./configure \
            --prefix=$PWD/inst \
            --enable-cassert --enable-debug --enable-tap-tests --enable-rpath \
            --with-tcl --with-python --with-gssapi --with-pam --with-ldap \
            --with-openssl --with-libedit-preferred --with-uuid=e2fs \
            --with-ossp-uuid  --with-libxml --with-libxslt  --with-perl \
            --with-icu --with-libnuma --enable-injection-points
```




### PGroonga version

pgroonga: (HEAD detached at 4.0.4)

### Groonga version

groonga --version
```
Groonga 15.0.4 [linux-gnu,x86_64,utf8,match-escalation-threshold=0,nfkc,mecab,message-pack,mruby,onigmo,zlib,lz4,zstandard,epoll,rapidjson,xxhash]

configure options: < '--build=x86_64-linux-gnu' '--prefix=/usr' '--includedir=${prefix}/include' '--mandir=${prefix}/share/man' '--infodir=${prefix}/share/info' '--sysconfdir=/etc' '--localstatedir=/var' '--disable-option-checking' '--disable-silent-rules' '--libdir=${prefix}/lib/x86_64-linux-gnu' '--runstatedir=/run' '--disable-maintainer-mode' '--disable-dependency-tracking' '--with-munin-plugins' '--enable-mruby' '--disable-static' '--enable-document' '--with-pcre2' 'build_alias=x86_64-linux-gnu' 'CFLAGS=-g -O2 -Werror=implicit-function-declaration -ffile-prefix-map=/build/reproducible-path/groonga-15.0.4+dfsg=. -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection' 'LDFLAGS=-Wl,-z,relro -Wl,-z,now' 'CPPFLAGS=-Wdate-time -D_FORTIFY_SOURCE=2' 'CXXFLAGS=-g -O2 -ffile-prefix-map=/build/reproducible-path/groonga-15.0.4+dfsg=. -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection'>
```


### Additional context

_No response_

--- a/src/pgrn-custom-scan.c
+++ b/src/pgrn-custom-scan.c
@@ -781,8 +781,7 @@ PGrnMakeTupleTableSlot(CustomScanState *customScanState,
 	ListCell *cell;
 	ExecClearTuple(slot);
 	econtext->ecxt_scantuple = slot;
-	Snapshot snapshot = SnapshotAny;
-	snapshot = RegisterSnapshot(GetTransactionSnapshot());
+	Snapshot snapshot = RegisterSnapshot(GetTransactionSnapshot());
 	// We might add state->targetlist instead of ss.ps.plan->targetlist.
 	foreach (cell, customScanState->ss.ps.plan->targetlist)
 	{
@@ -857,8 +856,7 @@ PGrnMakeTupleTableSlot(CustomScanState *customScanState,
 		}
 		ttsIndex++;
 	}
-	if (snapshot != SnapshotAny)
-		UnregisterSnapshot(snapshot);
+	UnregisterSnapshot(snapshot);
 	return ExecStoreVirtualTuple(slot);
 }
 

